qa/python_models/sub_add/model.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
It strange to have both add_sub and sub_add when they are nearly identical. There is a better way to do this that we can discuss.